apps/admin/frontend/src/components/format_usb_modal.tsx

```diff
@@ -82,8 +82,8 @@ function FormatUsbFlow({ onClose }: FormatUsbModalProps): JSX.Element {
           title="Confirm Format USB Drive"
           content={
             <P>
-              <Font weight="bold">Warning:</Font> formatting will delete all
-              files on the USB drive. Back up USB drive files before formatting.
+              <Icons.Warning color="warning" /> Formatting will delete all files
+              on the USB drive. Back up USB drive files before formatting.
```

Comment:
<img width="550" alt="Screen Shot 2024-10-08 at 12 33 53 PM" src="https://github.com/user-attachments/assets/9b4b7af1-0531-46e6-9b5b-6b9201ea62a1">
<img width="545" alt="Screen Shot 2024-10-08 at 12 34 01 PM" src="https://github.com/user-attachments/assets/f391011f-5b0b-4879-8a0e-18ffe965193f">

Reply:
Maybe a danger button instead of primary here?